modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -201,6 +201,7 @@ ContentDashboardAdminManagementToolbarDisplayContext contentDashboardAdminManage
 
 							<liferay-ui:search-container-column-text>
 								<clay:dropdown-actions
+									dropdownCssClass="dropdown-action"
```

Comment:
This looks like something that we shouldn't need to specify on each Tag use since according to clay markup all actions dropdowns should have that css class.

Reply:
So, you mean it's better to add in directly in `ClayDropDownWithItems` component?

```js
className="dropdown-action"
```

Reply:
I saw that the component itself it's not adding `dropdown-action` by default (only dropdown).

Reply:
> So, you mean it's better to add in directly in ClayDropDownWithItems component in DropdownMenu.js file?

Nope. `DropdownMenu.js` is the component that is being rendered when you use `clay-dropdown-menu` and `dropdown-actions`, that's why I suggested [this three options](https://github.com/liferay-frontend/liferay-portal/pull/211#issuecomment-671209332). Based on your approach I'd go for the second one.